src/App.js

```diff
@@ -3,19 +3,23 @@ import "./App.css";
 import HomePage from "./components/HomePage";
 import Nav from "./components/Nav";
 import BookPage from "./components/BookPage";
-import ClaimedBookPage from "./components/ClaimedBookPage";
-import AddBook from "./components/Nav/AddBook";
+// import ClaimedBookPage from "./components/ClaimedBookPage";
+import AddBook from "./components/AddBook";
+import { useState } from "react";
 
 function App() {
+  const [claimedUrl, setClaimedUrl] = useState(
+    "https://book-swap-api.dev.io-academy.uk/api/books?claimed=0"
+  );
+
   return (
     <div className="App">
       <BrowserRouter>
-        <Nav />
+        <Nav setClaimedUrl={setClaimedUrl} claimedUrl={claimedUrl} />
 
         <Routes>
           <Route path="/book/:id" element={<BookPage />} />
-          <Route path="/" element={<HomePage />} />
-          <Route path="/claimed/" element={<ClaimedBookPage />} />
+          <Route path="/" element={<HomePage claimedUrl={claimedUrl} />} />
```

Comment:
You should still have two routes for / and claimed.

Rather than passing in the entire URL using this system shared between app and the Nav, it's better to just have two routes that both display the HomePage (you should rename it though, it's not just the homepage anymore) but pass in a prop that sets whether or not you want claimed books.